index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Product Landing Page</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header id="header">
+        <img src="C:\Users\Black Store\Pictures\s.png" alt="Logo" id="header-img">
+        <nav id="nav-bar">
+          <a href="#features" class="nav-link">Features</a>
+          <a href="#how-it-works" class="nav-link">How It Works</a>
+          <a href="#pricing" class="nav-link">Pricing</a>
+          <a href="#contact" class="nav-link">Contact</a>
+        </nav>
+    </header>
```

Comment:
- Your code uses an indentation of 4 spaces instead it should use an indentation of 2 spaces.